infrastructure/deployment/deploy.sh

```diff
@@ -289,9 +289,11 @@ docker_stack_deploy() {
     do
       echo "Server failed to download $tag. Retrying..."
       sleep 5
-    done
+    done &
   done
-
+  echo "Images are getting downloaded"
```

Comment:
Would this disappear in the log output of the pull(s)? Eg are they outputting earlier and parallel so this gets lost in their log output...?

If so we can drop this

Reply:
No I could see it as i have rephrased the sentence "Images download is in process" to "Images are getting downloaded" if you check https://github.com/opencrvs/opencrvs-farajaland/actions/runs/10938830700/job/30369325656 you could see it after downloaded the images